gateway/ln-gateway/src/lightning/mod.rs

```diff
@@ -120,9 +138,13 @@ pub trait ILnRpcClient: Debug + Send + Sync {
         .await
     }
 
-    /// Attempt to pay an invoice using the lightning node using a
+    /// Attempts to pay an invoice using the lightning node using a
     /// [`PrunedInvoice`], increasing the user's privacy by not sending the
-    /// invoice description to the gateway.
+    /// invoice description to the gateway. If this is implemented, then
```

Comment:
"using a invoice description" - this reads a little awkwardly to me, maybe rephrase